lib/sla.js

```diff
@@ -1,6 +1,77 @@
-'use strict';
+/*jslint node: true */
+'use strict'
 
-module.exports = function(callback){
-    console.log("\"snarflegluff\" should be printed below:");
-    var str = "snarflegluff";
-    return str;}
+/**
+ * Looks for an existing mixing bridge and creates one if none exist.
+ */
+function designateMixingBridge(client, callback) {
+  client.bridges.list(function(err, bridges) {
+    var bridge = bridges.filter(function(candidate) {
+      return candidate.bridge_type === 'mixing';
```

Comment:
For this current task, yes, that's the expected behaviour. Later on, they'll have to be a bit smarter about how they select the bridges.